main.cpp

```diff
@@ -108,7 +108,52 @@ struct CarWash
     You'll need to insert the Person struct from the video in the space below.
  */
 
+struct Person
+{
+    int age;
+    bool leftHanded;
+    float distanceTravled;
+    float leftlegLength;
+    float leftfootLength;
+    float rightlegLength;
+    float rightfootLength;
+
+    struct Leg
+    {
+        void stepForward();
+        float stepSize(float legLength, float footLength);
+    };
 
+    void run(int howFast, bool leftFootFirst);
+};
+
+float Person::Leg::stepSize(float legLength, float footLength)
+{
+    float stepSize = legLength * footLength;
+    return stepSize;
+}
+
+void Person::Leg::stepForward()
+{
+}
+
+void Person::run(int howFast, bool leftLegFirst)
+{
+    Leg leftLeg;
+    Leg rightLeg;
+
```

Comment:
These should be member variables in `Person`
A person's legs aren't created every time they run..
- [ ] revise

Reply:
Got it! Yeah that makes sense.